core/tests/karma.conf.js

```diff
@@ -1,32 +1,26 @@
+var generatedJs;
+process.argv.forEach(function(argument) {
+  if (argument == 'minify') {
+    generatedJs = 'third_party/generated/prod/js/third_party.js';
+  } else {
```

Comment:
I have found a better way to deal with this. we used ```yargs``` npm module to format command line args in gulpfile. I used the same package to in karma.connf.js. Now with yargs we can use to find value of a specific argument like ```minify=True``` just by ```argv.minify``` and it will return ```True```. here args are treated as key to value dict type.so It will help to solve your concern also about  if we add other more args.